src/vue/detailFilm.php

```diff
@@ -1,67 +1,110 @@
-<div class="card w-auto mb-3 my-2 mx-2" data-bs-theme="dark">
-    <div class="row g-0">
-        <div class="col-md-6 mt-1">
-            <img src="static/img/<?= $detailFilm->id_affiche ?>.jpg" class="img-fluid rounded-start" alt="...">
-        </div>
-        <div class="col-md-6">
-            <div class="card-body">
-                <h5 class="card-title"><?= $detailFilm->titre ?></h5>
-                <hr>
-                <p class="card-text"><?= $detailFilm->description ?></p>
-                <p class="card-text"><small class="text-body-secondary"><?= $detailFilm->duree ?></small></p>
-                <hr>
-                <?php
-                if (count($detailFilm->seances) !== 0) {
-                ?>
-                    <form id="formulaire-cinema" action="?action=achat" method="post">
-                        <div class="d-flex d-inline-block col-md-6">
-                            <select class="form-select col-md-3" id="ville-select" name="ville-select" aria-label="Default select example">
-                                <option value="" selected disabled hidden>Selectionner une ville</option>
-                                <?php
-                                foreach ($detailFilm->villes as $key => $ville) {
-                                ?>
-                                    <option value=<?= $ville->id ?>><?= $ville->nom ?></option>
-                                <?php
-                                }
-                                ?>
-                            </select>
-                            <select class="form-select col-md-3" id="cinema-select" name="cinema-select" aria-label="Default select example">
-                            </select>
-                        </div>
-                        <div class="btn-group d-inline-block my-1" id="seance-select" name="seance" role="group" aria-label="Basic radio toggle button group">
-                        </div>
-                        <div class="d-flex d-inline-block">
-                            <label class="form-label col-md-4" id="place-select-label">Nombre de places</label>
-                            <select class="form-select" id="place-select" name="place-select" aria-label="Default select example">
-                                <option value="" selected disabled hidden>Selectionner</option>
-                                <?php
-                                for ($i = 0; $i < 10; $i++) {
-                                ?>
-                                    <option value=<?= $i + 1 ?>><?= $i + 1 ?></option>
-                                <?php
-                                }
-                                ?>
-                            </select>
-                        </div>
+<?php if ($detailFilm->seances == null) { ?>
```

Comment:
Moi ca ne marchait pas au début parceque certains films en streaming avait des séances. 
C'est plus sûr de rajouter l'état dans detailFilm :
```suggestion
<?php if ($detailFilm->etat == 'streaming') { ?>
```